src/components/common/ChangeLocationButton.js

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+
+function ChangeLocationButton({ TextCL, DItem1, DItem2, DItem3, type }) {
```

Comment:
Everything else besides class names and component names is camelCase in JS. 
See: https://www.w3schools.com/js/js_conventions.asp

Reply:
Fixed camelCase in project